### PR TITLE
Feature/export bidders

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Also set up the redux and actions, just to have the framework ready to go.

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Will fix tests tomorrow.

File: src/actions/bureauPositions.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { batch } from 'react-redux';
+import { downloadFromResponse } from 'utilities';
+import { toastError } from './toast';
+import api from '../api';
+
+// eslint-disable-next-line import/prefer-default-export
+export function downloadBidderData() {
+  // exporting the 'bureau positions'. Just setting up the framework
+  const url = '/fsbid/bureau/positions/export/';
+  return api().get(url, {
```
Comment (mjoyce91) [inline src/actions/bureauPositions.js:10]: Take a look at how we do this elsewhere - Favorites will probably be a good example

File: src/actions/bureauPositions.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+export function bureauPositionsFetchData() {
+  return (dispatch) => {
+    dispatch(bureauPositionsIsLoading(true));
+    const url = '/fsbid/bureau/positions/';
+    api().get(url)
+      .then(({ data }) => {
+        dispatch(bureauPositionsIsLoading(false));
+        dispatch(bureauPositionsHasErrored(false));
+        dispatch(bureauPositions(data));
```
Comment (mjoyce91) [inline src/actions/bureauPositions.js:52]: isLoading should be last

File: src/actions/bureauPositions.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      })
+      .catch(() => {
+        batch(() => {
+          dispatch(bureauPositionsIsLoading(false));
```
Comment (mjoyce91) [inline src/actions/bureauPositions.js:56]: isLoading should be last

File: src/Components/BureauPage/PositionManagerDetails/PositionManagerDetails.jsx [hunk 4]
```diff
@@ -1,38 +1,55 @@
...
-  */
+  exportBidders = () => {
+    this.setState({ isLoading: true });
+    downloadBidderData()
+      .then(() => {
+        this.setState({ isLoading: false });
+      })
+      .catch(() => {
+        this.setState({ isLoading: false });
+      });
```
Comment (mjoyce91) [inline src/Components/BureauPage/PositionManagerDetails/PositionManagerDetails.jsx:27]: I'm not sure this is going to do what you expect it to. The downloadBidderData function is already handling the then/catch, so these won't do anything.

Comment (elizabeth-jimenez) [reply src/Components/BureauPage/PositionManagerDetails/PositionManagerDetails.jsx:27]: lol, oh! I forgot to switch this out for `bureauPositionsIsLoading` once I got it set up 😅  Thanks Mike!

**Comment by elizabeth-jimenez** [comment]
Export loading not updating locally. Need to update.

File: src/Components/BureauPage/PositionManager/PositionManager.jsx [hunk 5]
```diff
@@ -1,17 +1,25 @@
...
 import ResultsControls from '../../ResultsControls/ResultsControls';
 
 
-const PositionManager = () => {
+const PositionManager = props => {
   const [textValue, setTextValue] = useState('temp text');
+  // eslint-disable-next-line no-unused-vars
+  const [page, setPage] = useState(1);
+  // eslint-disable-next-line no-unused-vars
+  const [sortType, setSortType] = useState();
```
Comment (mjoyce91) [inline src/Components/BureauPage/PositionManager/PositionManager.jsx:21]: are these eslint disables just temporary until we have real data?

File: src/actions/bureauPositions.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
+import { batch } from 'react-redux';
+import { downloadFromResponse } from 'utilities';
+import { toastError } from './toast';
+import api from '../api';
+
+// eslint-disable-next-line import/prefer-default-export
+export function downloadBidderData() {
+// export function downloadBidderData(id) {
+  // const url = createUrl(`/fsbid/bureau/positions/${id}/bidders/export/`);
+  // exporting the 'bureau positions'. Just setting up the framework, for now.
```
Comment (mjoyce91) [inline src/actions/bureauPositions.js:10]: 👍

File: src/Components/BureauPage/PositionManager/PositionManager.jsx (hunk 5 above)
Comment (elizabeth-jimenez) [reply src/Components/BureauPage/PositionManager/PositionManager.jsx:21]: Yep!

**Pull request merged**